crawl-ref/source/dat/des/variable/large_themed.des

```diff
@@ -2316,3 +2316,85 @@ xxxx-xx..xx....xx-xxx-x
 xxxxxxxxx-x.....xxxxxxx
 xxxxxxxxxxx.@@@..xxxxxx
 ENDMAP
+
+NAME:   dreamdust_cave_troll
+TAGS:   no_monster_gen
+ORIENT: northwest
+DEPTH:  D:10-, Lair
+NSUBST: - = 1 / 2=? / .
+KFEAT:  ^ = web
+MONS:   deep troll
+: dgn.delayed_decay(_G, "?", "yak corpse")
+MARKER: ! = lua:fog_machine { cloud_type='flame', pow_min = 10, pow_max = 10, \
+                              delay = 10, size = 1, excl_rad = 0 }
+MAP
+xxxxxxxxxx
+x--xxxx
+x-!-xxxx
+xx-.xxxxx
+xxxx.^xxx
+xxxxx.xx
+xxx^.xx
+xxx@xx
+x
+ENDMAP
+
+NAME:   dreamdust_under_siege
+TAGS:   transparent no_monster_gen no_pool_fixup
+ORIENT: south
+DEPTH:  D:12-, Orc:1, Depths, !Depths:1
+WEIGHT: 5
+TILE:   c = wall_stone_smooth
+FTILE:  +'{([>-O = floor_cobble_blood
+# Monster Placement
+NSUBST: ' = 6=1 / 6=1' / '
+NSUBST: . = 10=2 / 10=2. / 10=3 / 10=3. / 2=4 / 3=4. / .
+# Theme variations
+: if you.in_branch("D") then
+TILE:   G : dngn_statue_angel
+KFEAT:  O = altar_zin / altar_elyvilon / altar_the_shining_one
+FTILE:  -O : floor_limestone / floor_w_marble
+MONS:   vault guard, human zombie / ogre zombie, \
+        human skeleton / ogre skeleton, necromancer
+: elseif you.in_branch("Orc") then
+SUBST:  ( = {, [ = >, G = I
+KFEAT:  > = escape_hatch_down
+KFEAT:  O = altar_beogh
+KPROP:  -O = bloody
+MONS:   orc / orc warrior, orc zombie / ogre zombie, \
+        orc skeleton / ogre skeleton, necromancer
+: else
+# Depths
+:    if crawl.coinflip() then
+# Holy defenders
+TILE:   G : dngn_statue_angel
+KFEAT:  O = altar_zin / altar_elyvilon / altar_the_shining_one
+FTILE:  -O : floor_limestone / floor_w_marble
+MONS:   angel / vault guard, zombie, simulacrum, lich / ancient lich w:3
```

Comment:
I'd recommend simply using a mix of cherubs, angels, and daevas as defenders instead of having vault guards. It's fine to use any holy monsters you feel are appropriate, including the possibility of a pearl dragon "boss". Those are all going to be something a player in Depths can handle.

For the "attacking" monsters, I recommend mixing in some spectres with the simulacra. It actually works best if you look at the depths monster set and hand pick interesting zombies, simulacra, spectres. For zombies, fast monsters work nicely, or at least monsters with heavy melee attacks. Many random zombies you'll get will be functionally useless, e.g. spriggan air mage zombies. For simulacra and spectres, there's a similar problem of some being useless or nearly so. Fast and heavy or multiple attacks work well. 

If you want a feel that's more a mix of relative popcorn, it's fine to just use a simple `zombie` or `simulacra` and let it be random, but I would recommend at least mixing in some of `spectre`, since those don't lose monster speed as derived undead.